packages/config/config/devices/0x027a/zen25.json

```diff
@@ -55,54 +54,53 @@
 		},
 		{
 			"#": "3",
-			"label": "Wattage Frequency",
-			"unit": "Seconds",
+			"label": "Power (W) Report Frequency",
+			"unit": "seconds",
 			"valueSize": 4,
 			"minValue": 30,
 			"maxValue": 2678400,
 			"defaultValue": 30
 		},
 		{
 			"#": "4",
-			"label": "Energy Frequency",
-			"unit": "Seconds",
+			"label": "Energy (kWh) Report Frequency",
+			"unit": "seconds",
 			"valueSize": 4,
-			"minValue": 30,
+			"minValue": 5,
 			"maxValue": 2678400,
 			"defaultValue": 300
 		},
 		{
 			"#": "5",
-			"label": "Voltage Frequency",
-			"unit": "Seconds",
+			"label": "Voltage (V) Report Frequency",
+			"unit": "seconds",
 			"valueSize": 4,
-			"minValue": 30,
+			"minValue": 5,
 			"maxValue": 2678400,
 			"defaultValue": 300
 		},
 		{
 			"#": "6",
-			"label": "Current Frequency",
+			"label": "Current (A) Report Frequency",
 			"description": "Electrical Current (A) Report Frequency",
-			"unit": "Seconds",
+			"unit": "seconds",
 			"valueSize": 4,
-			"minValue": 30,
+			"minValue": 5,
 			"maxValue": 2678400,
 			"defaultValue": 300
 		},
 		{
 			"#": "7",
 			"label": "Overload Protection",
-			"unit": "Amps",
+			"unit": "A",
 			"valueSize": 1,
 			"minValue": 1,
 			"maxValue": 10,
 			"defaultValue": 10
 		},
 		{
 			"#": "8",
-			"label": "Enable Auto-Off (Left)",
-			"description": "Enable Auto Turn-Off Timer for Left Outlet",
+			"label": "Enable Auto-Off (Left Outlet)",
```

Comment:
We try to avoid `Enable` and `Disable` in labels, especially when the only options are `Enable/Disable`. Suggestion:
Param 8: `Left Outlet: Auto-Off`
Param 9: `Left Outlet: Auto-Off Timer` (or `... Delay`)
Param 10: `Left Outlet: Auto-On`
Param 11: `Left Outlet: Auto-On Timer` (or `... Delay`)
...and similar for the right outlet.

Reply:
Done, I was wondering about this one. Should I clarify it in the style guide?

Reply:
Yes, please!